src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs

```diff
@@ -138,7 +138,7 @@ private ViewVacancy GetDisplayViewModelForClosedVacancy(Vacancy vacancy)
         private void PopulateViewModelWithApplications(Vacancy vacancy, DisplayVacancyApplicationViewModel viewModel)
         {
             var mappedDisplayVacancyViewModelTask = _vacancyDisplayMapper.MapFromVacancyAsync(viewModel, vacancy);
-            var vacancyApplicationsTask = _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
+            var vacancyApplicationsTask = _vacancyClient.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
```

Comment:
Again. Can we remove the IEmployerVacancyClient _client from the constructor?

Reply:
👍